routeguide/client/src/main/scala/handlers/RouteGuideClientHandler.scala

```diff
@@ -138,7 +140,7 @@ class RouteGuideClientHandler[F[_]: ConcurrentEffect](
             .map { note: RouteNote =>
               logger.info(
                 s"Got message '${note.message}' at " +
-                  s"${note.location.latitude}, ${note.location.longitude}"
+                  s"${note.location.get.latitude}, ${note.location.get.longitude}"
```

Comment:
There are still a few `.get` ^^

Reply:
ooops, sorry!  I should have them all now